docs/book/en/src/embed/go/ref.md

```diff
@@ -134,105 +160,139 @@ pub fn obfusticate(s: String) -> String {
+rustup target add wasm32-wasi
 cd rust_bindgen_funcs
-rustwasmc build
-# The output WASM will be pkg/rust_bindgen_funcs_lib_bg.wasm
+cargo build --target wasm32-wasi --release
+# The output WASM will be target/wasm32-wasi/release/rust_bindgen_funcs_lib.wasm
 ```
 
-The [Golang source code](https://github.com/second-state/WasmEdge-go-examples/blob/master/go_BindgenFuncs/bindgen_funcs.go) to run the WebAssembly function in WasmEdge is as follows. The `ExecuteBindgen()` function calls the WebAssembly function and passes the parameters with the `wasm-bindgen` supporting.
+The [Golang source code](https://github.com/second-state/WasmEdge-go-examples/blob/master/wasmedge-bindgen/go_BindgenFuncs/bindgen_funcs.go) to run the WebAssembly function in WasmEdge is as follows. The `bg.Execute()` function calls the WebAssembly function and passes the parameters with the `wasmedge-bindgen` supporting.
 
 ```go
 package main
 
 import (
-    "fmt"
-    "os"
+	"fmt"
+	"os"
 
-    "github.com/second-state/WasmEdge-go/wasmedge"
+	"github.com/second-state/WasmEdge-go/wasmedge"
+	bindgen "github.com/second-state/wasmedge-bindgen/host/go"
 )
 
 func main() {
-    // Expected Args[0]: program name (./bindgen_funcs)
-    // Expected Args[1]: wasm or wasm-so file (rust_bindgen_funcs_lib_bg.wasm)
-
-    wasmedge.SetLogErrorLevel()
-
-    var conf = wasmedge.NewConfigure(wasmedge.WASI)
-    var vm = wasmedge.NewVMWithConfig(conf)
-    var wasi = vm.GetImportObject(wasmedge.WASI)
-    wasi.InitWasi(
-        os.Args[1:],     // The args
-        os.Environ(),    // The envs
-        []string{".:."}, // The mapping directories
-    )
-
-    // Instantiate wasm
-    vm.LoadWasmFile(os.Args[1])
-    vm.Validate()
-    vm.Instantiate()
-
-    // Run bindgen functions
-    var res interface{}
-    var err error
-    
-    res, err = vm.ExecuteBindgen("say", wasmedge.Bindgen_return_array, []byte("bindgen funcs test"))
-    if err == nil {
-        fmt.Println("Run bindgen -- say:", string(res.([]byte)))
-    } 
-    res, err = vm.ExecuteBindgen("obfusticate", wasmedge.Bindgen_return_array, []byte("A quick brown fox jumps over the lazy dog"))
-    if err == nil {
-        fmt.Println("Run bindgen -- obfusticate:", string(res.([]byte)))
-    } 
-    res, err = vm.ExecuteBindgen("lowest_common_multiple", wasmedge.Bindgen_return_i32, int32(123), int32(2))
-    if err == nil {
-        fmt.Println("Run bindgen -- lowest_common_multiple:", res.(int32))
-    } 
-    res, err = vm.ExecuteBindgen("sha3_digest", wasmedge.Bindgen_return_array, []byte("This is an important message"))
-    if err == nil {
-        fmt.Println("Run bindgen -- sha3_digest:", res.([]byte))
-    } 
-    res, err = vm.ExecuteBindgen("keccak_digest", wasmedge.Bindgen_return_array, []byte("This is an important message"))
-    if err == nil {
-        fmt.Println("Run bindgen -- keccak_digest:", res.([]byte))
-    } 
-
-    vm.Release()
-    conf.Release()
+	/// Expected Args[0]: program name (./bindgen_funcs)
```

Comment:
Please help to modify all `///` to `//`. Thanks.

Reply:
Done

Reply:
Done